packaging/windows-package.sh

```diff
@@ -23,11 +26,11 @@ if [[ ! "${ZIPDIR}" ]]; then
 fi
 
 # Download sshfs-win.msi + winfsp.msi
-curl -L -o "${ZIPDIR}"/winfsp.msi https://github.com/billziss-gh/winfsp/releases/download/v1.9/winfsp-1.9.21096.msi
-curl -L -o "${ZIPDIR}"/sshfs-win.msi https://github.com/billziss-gh/sshfs-win/releases/download/v3.7.21011/sshfs-win-3.7.21011-x64.msi
+curl -L -o "${ZIPDIR}"/winfsp.msi "https://github.com/billziss-gh/winfsp/releases/download/v1.9/winfsp-${WINFSP_VERSION}.msi"
```

Comment:
You can add the 1.9 by doing `${WINFSP_VERSION%.*}` -- that will chomp off the last . and everything after it

Reply:
Ah thank you! you learn something new in bash every day